### PR TITLE
Fixes #8943 - Update release notes for 2.2.6

### LINKED ISSUE #8943: Enable the predictive intellisense by default in PSReadLine
### Prerequisites

- [X] Write a descriptive title.
- [X] Search the existing issues.

### Summary of the new feature or changed behavior

Currently, the predictive intellisense feature is disabled by default in PSReadLine. The PR https://github.com/PowerShell/PSReadLine/pull/3351 is going to conditionally enable this feature by default.

- when VT is not supported, `PredictionSource` is set to `None` by default
- when VT is supported and it's running with PowerShell 7.2+, `PredictionSource` is set to `HistoryAndPlugin` by default
- when VT is supported and it's running with PowerShell prior to 7.2, `PredictionSource` is set to `History` by default

### List of articles that need to be updated

https://docs.microsoft.com/en-us/powershell/module/psreadline/set-psreadlineoption?view=powershell-7.2

### Link to related PR in PowerShell/PowerShell repo

- https://github.com/PowerShell/PSReadLine/pull/3351

### Link to related Issues in PowerShell/PowerShell repo

- https://github.com/PowerShell/PSReadLine/pull/3351

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes AB#1964866 - Fixes #8943 - Update release notes for 2.2.6

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://docs.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/7.3/PSReadLine/About/about_PSReadLine.md [hunk 1]
```diff
@@ -82,6 +79,13 @@ To disable Predictive IntelliSense, just run:
 Set-PSReadLineOption -PredictionSource None
 ```
 
+> [!NOTE]
+> Predictive IntelliSense is enabled by default in PSReadLine 2.2.6. For more
+> information see, the [PSReadLine release history](#psreadline-release-history)
+> in the **Notes** section below.
+
+## Using the **Microsoft.PowerShell.PSConsoleReadLine** class
```
Comment (michaeltlombardi) [inline reference/7.3/PSReadLine/About/about_PSReadLine.md:87]:
```suggestion
## Using the Microsoft.PowerShell.PSConsoleReadLine class
```

**Review by michaeltlombardi — DISMISSED** [review]
Minor cleanup but content is 👍🏻

File: reference/5.1/PSReadLine/About/about_PSReadLine.md [hunk 2]
```diff
@@ -15,7 +15,12 @@ PowerShell console.
 
 ## Long Description
 
-PSReadLine 2.0 provides a powerful command-line editing experience for the
+PowerShell 5.1 ships with PSReadLine 2.0.0. There are newer versions
+available. The current version of PSReadLine can be installed and used on
+Windows PowerShell 5.1 and newer. For some features, you need running
```
Comment (michaeltlombardi) [inline reference/5.1/PSReadLine/About/about_PSReadLine.md:20]:
```suggestion
Windows PowerShell 5.1 and newer. For some features, you need to be running
```

File: reference/5.1/PSReadLine/About/about_PSReadLine.md [hunk 3]
```diff
@@ -15,7 +15,12 @@ PowerShell console.
 
 ## Long Description
 
-PSReadLine 2.0 provides a powerful command-line editing experience for the
+PowerShell 5.1 ships with PSReadLine 2.0.0. There are newer versions
+available. The current version of PSReadLine can be installed and used on
+Windows PowerShell 5.1 and newer. For some features, you need to be running
+PowerShell 7.1 or higher.
```
Comment (daxian-dbw) [inline reference/5.1/PSReadLine/About/about_PSReadLine.md:21]: Should be 7.2 or higher.

File: reference/7.2/PSReadLine/About/about_PSReadLine.md [hunk 4]
```diff
@@ -1731,7 +1728,63 @@ example, the user is reading help to decide which parameter to use.
...
+- **PSReadLine 2.2.5**
+
+  This release rolls up the following enhancements added since the 2.1.0
+  release:
+
+  - PSReadLine added two new predictive IntelliSense features:
+    - Added the **PredictionViewStyle** parameter to allow for the selection of
+      the new `ListView`.
+    - Connected PSReadLine to the `CommandPrediction` APIs introduced in
+      PowerShell 7.1 to allow a user can import a predictor module that can
```
Comment (daxian-dbw) [inline reference/7.2/PSReadLine/About/about_PSReadLine.md:1764]: Should be 7.2

Comment (sdwheeler) [reply reference/7.2/PSReadLine/About/about_PSReadLine.md:1764]: This change was introduced in 7.1.

File: reference/5.1/PSReadLine/About/about_PSReadLine.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.1/PSReadLine/About/about_PSReadLine.md:21]: I will change the recommendation to 7.2 here.

File: reference/7.2/PSReadLine/About/about_PSReadLine.md (hunk 4 above)
Comment (daxian-dbw) [reply reference/7.2/PSReadLine/About/about_PSReadLine.md:1764]: This change was done in 7.2. You can import predictor module only in 7.2

**Pull request merged**